### ISSUE TITLE
Support assigning sandboxes to jobs

### ISSUE THREAD

**Issue by chrisburr** [DIRACGrid/diracx#143]
Add routes to the [Sandbox handler](https://github.com/DIRACGrid/diracx/blob/main/src/diracx/routers/job_manager/sandboxes.py)

* Getting input and output sandboxes at the sane time:

```json
GET /api/jobs/{job_id}/sandbox
{
    "Input": {pfn1: presignedurl, pfn2: presignedurl},
    "Output": {pfn1: presignedurl, pfn2: presignedurl},
}
```

* Get just the input or output sandboxes:

```json
GET /api/jobs/{job_id}/sandbox/{sandbox_type}
{pfn1: presignedurl, pfn2: presignedurl}
```

* Assign an output sandbox (it's not currently forseen there is a need to assign input sandboxes)

```json
PATCH "SE:SandboxSE|/S3/.../abcedfg.tar.bz2" /api/jobs/{job_id}/sandbox/output
```

Part of #13

**Comment by chrisburr** [comment]
@natthan-pigoux is taking a look at this

**Comment by natthan-pigoux** [comment]
#156

**Comment by natthan-pigoux** [comment]
This issue can be closed since #156 has been merged.
